ruamel_yaml/meta.yaml

```diff
@@ -2,17 +2,19 @@ package:
   name: ruamel_yaml
   version: 0.11.14
 
+source:
+  fn: ruamel_yaml-0.11.14.tar.gz
+  url: https://bitbucket.org/ruamel/yaml/get/0.11.14.tar.gz
+  sha256: b78d394e192b6d0911bd59f794ceea88143fac4e373b76ed7e48143a2a29b33c
+
 build:
   number: 0
   no_link: .*\.(pyd|dll)                       [win]
 
-source:
-  hg_url: https://bitbucket.org/ruamel/yaml
-  hg_tag: 0.11.14
-
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
As we run [`python setup.py --version`]( https://github.com/ContinuumIO/anaconda-recipes/blob/293f1170d2433b9ef87c148400ad39e7d5507f08/ruamel_yaml/prepare.bash#L7 ) to get the version before patching, we need to meet `ruamel.yaml`'s requirements for `setup.py` to run. This means we need `setuptools`. Without this requirement, this command will fail and the `VERSION` environment variable will be empty.